mav_msgs/include/mav_msgs/conversions.h

```diff
@@ -106,6 +118,30 @@ inline void eigenOdometryFromMsg(const nav_msgs::Odometry& msg, EigenOdometry* o
   odometry->angular_velocity = mav_msgs::vector3FromMsg(msg.twist.twist.angular);
 }
 
+inline void eigenCommandTrajectoryPositionYawFromMultiDofJointTrajectoryPointMsg(
+    const trajectory_msgs::MultiDOFJointTrajectoryPoint& msg,
+    EigenCommandTrajectoryPositionYaw* command_trajectory) {
+  assert(command_trajectory != NULL);
+
+  if (msg.transforms.size() > 1) {
+    ROS_WARN("MultiDofJointTrajectoryPoint message should have one joint, but has %lu. Using first joint.",
+             msg.transforms.size());
+  }
+
+  if (msg.transforms.empty()) {
```

Comment:
move this check to before the other if, as this gets checked every time, but it can return directly without having to check the above if.